.github/workflows/downstream_protobuf_compatibility_check_nightly.yaml

```diff
@@ -0,0 +1,48 @@
+on:
+  pull_request:
+    # Updates on Release-Please PRs
+    branches:
+      - 'release-please--branches--main'
+  workflow_dispatch:
+    inputs:
+      protobuf_versions:
+        description: 'Comma separated list of Protobuf-Java versions (i.e. "3.25.x","4.x.y")'
+        required: true
+  schedule:
+    - cron: '0 1 * * *' # Nightly at 1am
+
+name: Downstream Protobuf Compatibility Check Nightly
+jobs:
+  downstream-protobuf-test:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        repo:
```

Comment:
Are all of them dependent on `protobuf-java`? For example, I don't see `java-storage-nio` is dependent on `protobuf-java`. Or we are adding all handwritten libraries here in case they start depending on `protobuf-java` in the future?

Reply:
I took a quick look and don't think they're all dependent on protobuf-java. java-storage-nio seems to be the odd one out that doesn't have any protobuf-java references at all and a few others have some references only in tests. 

> adding all handwritten libraries here in case they start depending on protobuf-java in the future

I think it'll be safer to keep all of the downstream libraries in here. Just to be safe in case there are references of Protobuf-Java added in source for downstream libraries.